persim/pers_landscape_transf.py

```diff
@@ -0,0 +1,69 @@
+"""
+    Implementation of scikit-learn transformers for persistence
+    landscapes.
+
+    authors: Gabrielle Angeloro, Michael Catanzaro
+"""
+from operator import itemgetter
+from sklearn.base import BaseEstimator, TransformerMixin
+from .pers_landscape_exact import PersLandscapeExact
+from .pers_landscape_approx import PersLandscapeApprox
+
+
+__all__ = ["PLE", "PLA"]
+
+
+class PLE(BaseEstimator, TransformerMixin):
```

Comment:
What does PLE stand for?

Reply:
`PLE` is the PersLandscapeExact transformer, similarly for `PLA`. We're open to suggestions on re-naming.